docs: updated to reflect recent structural changes in repo

mad0perator opened the pull request:
Documentation was edited to reflect the unification of packages in repo.  Let me know what you think, I'd like all the feedback I can get.  Thank you.

mad0perator committed:
Edited `SETUP.md` draft.

mad0perator committed:
Merge branch 'CabinDAO:main' into main

mad0perator committed:
Edited documentation to reflect the recent unifying of packages in Topo repo

dvargas92495 commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,7 @@
+
+Until then, be kind and happy collaborating!
+
+[*back*](./README.md)
```
There's another PR up for CONTRIBUTING.md, let's nuke this to avoid a merge conflict

mad0perator replied:
Affirmative

dvargas92495 commented on README.md:
```diff
@@ -1,57 +1,67 @@
+[3]: https://github.com/CabinDAO/design-system/blob/main/CONTRIBUTING.md "Contributing"
+[4]: https://stitches.dev/docs/tokens "Theme Tokens"
+[5]: https://storybook.js.org/docs/react/get-started/introduction "Storybook Documentation"
+[6]: https://github.com/CabinDAO/design-system/projects "Projects"
+[7]: https://github.com/CabinDAO/design-system/discussions "Discussions"
```
Let's swap these out with clarity links:
- Topo bounties instead of projects: https://app.clarity.so/cabin/view/b06b6487-3625-41d7-8d6e-2f7a1fd32061
- We'll have a forum feature on Clarity soon in place of discussions, let's just direct to the discord channel for now

dvargas92495 commented on README.md:
```diff
@@ -1,57 +1,67 @@
+We look forward to creating with you!
+
+[1]: https://github.com/CabinDAO/design-system/issues/new "Issues"
+[2]: https://github.com/CabinDAO/design-system/discussions "Discussions"
```
this seems to be repeated twice with `7`

dvargas92495 commented on README.md:
```diff
@@ -1,57 +1,67 @@
+1. Fork the Topo repository into your Github account.
+2. Clone the forked repo to your local system.
+3. Navigate to the newly created project directory.
+4. `yarn` or `npm i` will install the Topo package along with its dependencies.
```
technically just installs the package's dependencies.

dvargas92495 replied:
let's add a note on using Storybook to visually verify components

mad0perator replied:
Thank you for the clarification regarding package dependencies.

Do I resolve all of the above and then submit a pull request?

dvargas92495 replied:
let's iterate on this one, especially since there appears to be a merge conflict

mad0perator committed:
Delete CONDUCT.md

This was a temporary file as a placeholder.  Nuking to avoid conflicts with pre-existing pull-request per @vargas~

mad0perator committed:
Delete CONTRIBUTING.md

nuking due to pre-existing PR (see previous commit)

dvargas92495 commented:
merged changes in https://github.com/CabinDAO/Topo/commit/0c6df8274c2292917708c9a9c2c959748029af33
